source/lib.cpp

```diff
@@ -93,6 +93,18 @@ void GZ_displayLinkInfo() {
     }
 }
 
+void GZ_displayZombieHoverInfo() {
+    //Generates A and B button presses per second
+    char a_presses_str[8];
+    char b_presses_str[8];
+
+    tww_sprintf(a_presses_str, "A: %d", GZ_getAPressesPerWindow());
+    tww_sprintf(b_presses_str, "B: %d", GZ_getBPressesPerWindow());
+
+    Font::GZ_drawStr(a_presses_str, 454.f, 320.f, 0x00CC00FF, g_dropShadows);

```

Comment:
Instead of typing out the hex values for green and red here, I wanted to see if there was a library I could access with defined color constants, but the cursor code did not seem to call upon such a library nor could I access what colors the cursor code used from here, but I could be mistaken on this.

If I'm not, then I don't think it would be a bad idea to make a color library so that the colors we choose are consistent (we shouldn't have two different greens, for example).